backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -206,4 +206,15 @@ public Facility markFacilityAsDeleted(@Argument UUID facilityId, @Argument boole
     }
     return organizationService.markFacilityAsDeleted(facilityId, deleted);
   }
+
+  /**
+   * Method HARD DELETES an Okta group without touching any of the application organization data.
+   * THIS SHOULD ONLY BE USED TO CLEAN UP ORGS CREATED IN OKTA FOR E2E TESTS. DON'T USE THIS METHOD
+   * FOR ANY LIVE OKTA API CALLS
+   */
+  @Transactional
+  @MutationMapping
+  public Organization deleteOktaOrganization(@Argument String orgExternalId) {
```

Comment:
Should we rename this method to specifically call out that this should be used for e2e tests only? Like `deleteE2EOktaOrganization` or something like that so we don't rely only on someone double checking the docstring before using it